builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -670,7 +672,10 @@ def writeServerlessFile(config, environment_logical_id){
   sh "sed -i -- 's/\${file(deployment-env.yml):subnetIds}/${config['subnetIds'] }/g' serverless.yml"
```

Comment:
Are we plugging in these details (`subnetIds`, `securityGroupIds`) into service metadata for each of the services in the list?

Reply:
Also, these two lines should be executed only for services that need VPC integration. You can move these to `addVpcDetails()`.

Reply:
> Are we plugging in these details (`subnetIds`, `securityGroupIds`) into service metadata for each of the services in the list? Otherwise, these details will not be available in `config`.

Yes. https://github.com/tmobile/jazz-installer/blob/ecs_private/installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh#L90

Reply:
> Also, these two lines should be executed only for services that need VPC integration. You can move these to `addVpcDetails()`.

Done.